### PR TITLE
	RavenDB-8514 Fix NRE in administrator Javascript Console

### PR DESCRIPTION
**Pull request by fitzchak**

### CODE REVIEW

File: src/Raven.Server/Documents/DocumentDatabase.cs [hunk 1]
```diff
@@ -85,7 +85,7 @@ internal void HandleOnRecoveryError(object sender, RecoveryErrorEventArgs e)
 
         public DocumentDatabase(string name, RavenConfiguration configuration, ServerStore serverStore)
         {
-            Scripts = new ScriptRunnerCache(this);
+            Scripts = new ScriptRunnerCache(this, Configuration);
```
Comment (ppekrol) [inline src/Raven.Server/Documents/DocumentDatabase.cs:88]: I think that this should be local variable, the 'Configuration' is set few lines lower.

Comment (ppekrol) [reply src/Raven.Server/Documents/DocumentDatabase.cs:88]: The real issue here is that Scripts should be created last, after Configuration property is set

Comment (ppekrol) [reply src/Raven.Server/Documents/DocumentDatabase.cs:88]: No need to pass the configuration separately, just move the initialization after setting the Configuration property

Comment (fitzchak) [reply src/Raven.Server/Documents/DocumentDatabase.cs:88]: Done

Comment (fitzchak) [reply src/Raven.Server/Documents/DocumentDatabase.cs:88]: We pass the configuration separately for the case that database is null and the configuration is the serverStore's one.

**Pull request merged**